Firmware/fw_ver_17/firmware_update_17.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
```

Comment:
@RobertLucian and @CleoQc you should be able to run this to update your firmware on your GoPiGo2.